### PR TITLE
feat(app): add UI to each modal step in ot-3 pipette cal

### PR DESCRIPTION
**Pull request by jerader**
closes RLIQ-227 RLIQ-223 RLIQ-221

Blocked to merge until https://github.com/Opentrons/opentrons/pull/11618 merges

# Overview

Add the UI to each step modal in the ot-3 pipette calibration flow: `BeforeBeginning`, `AttachStem`, `DetachStem`, `Results`. Only adds the UI aspects so nothing is really wired up yet.

Screen Shot 2022-10-26 at 4 05 37 PM

Screen Shot 2022-10-26 at 4 05 49 PM

Screen Shot 2022-10-26 at 4 05 58 PM

Screen Shot 2022-10-26 at 4 06 09 PM

# Changelog

- add the UI for `BeforeBeginning`, `AttachStem`, `DetachStem`, `Results`, updates tests
- extends the `PipetteWizardStepProps` to include `goBack` prop.
- make `subHeader` prop optional in `SimpleWizardBody`
- make small UI changes to `GenericWizardTile`
- add probe image to `equipmentImages`
- fix naming in `GenericWizardTile` related components (story and test)
- add probe image and a attach/detach stem image that will be updated later

# Review requests

- with an OT-3 pipette on the robot, click on `Calibrate pipette offset` button in the overflow menu. Click through the flow. There should be 4 pages and they should each more or less follow the figma designs. The main things that aren't very finalized are the attach/detach stem images.

# Risk assessment

low

### CODE REVIEW

File: app/src/molecules/WizardRequiredEquipmentList/index.tsx [hunk 1]
```diff
@@ -29,7 +29,7 @@ export function WizardRequiredEquipmentList(
   const { equipmentList, footer } = props
 
   return (
-    <Flex flexDirection={DIRECTION_COLUMN}>
+    <Flex flexDirection={DIRECTION_COLUMN} width="100%">
```
Comment (b-cooper) [inline app/src/molecules/WizardRequiredEquipmentList/index.tsx]: Is this `width="100%"` really what we'll want in every place that this component might be rendered? "Yes" might be an acceptable answer, though I'd lean towards having this component's props `extend StyleProps` that an instance of this component can spread whatever StyleProps it wants into the outer most container for specific layout purposes.

File: app/src/organisms/Devices/PipetteCard/PipetteOverflowMenu.tsx [hunk 2]
```diff
@@ -71,7 +71,7 @@ export const PipetteOverflowMenu = (
           </MenuItem>
         ) : (
           <>
-            {!isOT3PipetteAttached && (
+            {
```
Comment (b-cooper) [inline app/src/organisms/Devices/PipetteCard/PipetteOverflowMenu.tsx]: looks like these curly-braces are gratuitous now

File: app/src/organisms/PipetteWizardFlows/constants.ts [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  RESULTS: 'RESULTS',
+} as const
+
+export const FLOWS = {
+  ATTACH: 'ATTACH',
+  DETACH: 'DETACH',
+  CALIBRATE: 'CALIBRATE',
+}
+
+export const CALIBRATION_PROBE_DISPLAYNAME = 'Calibration Probe'
```
Comment (b-cooper) [inline app/src/organisms/PipetteWizardFlows/constants.ts]: nit: display name is two words elsewhere in the codebase. That would make this const `CALIBRATION_PROBE_DISPLAY_NAME`

**Comment by jerader** [comment]
TODO: address [this comment](https://github.com/Opentrons/opentrons/pull/11618#discussion_r1006147404) after that PR merges.

**Review by smb2268 — APPROVED** [review]
let's merge 🚀

**Pull request merged**